### PR TITLE
initial: Component structure with pnpm Monorepo

### PR DESCRIPTION
**Pull request by phibar**

### CODE REVIEW

File: Components/tla/EAM/Once/Cli/src/commands/hello/foo.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {Command} from '@oclif/core'
+
+export default class Foo extends Command {
```
Comment (mdonges) [inline Components/tla/EAM/Once/Cli/src/commands/hello/foo.ts]: i do not like the Foo command

**Review by mdonges — CHANGES_REQUESTED** [review]
well done

File: Components/tla/EAM/Once/Cli/src/commands/hello/foo.ts (hunk 1 above)
Comment (phibar) [reply Components/tla/EAM/Once/Cli/src/commands/hello/foo.ts]: done

**Review by mdonges — COMMENTED** [review]
well done ...no foo command

**Review by mdonges — APPROVED** [review]

**Pull request merged**